libs/components/lookup/src/lib/modules/lookup/lookup-autocomplete-adapter.ts

```diff
@@ -57,9 +58,8 @@ export class SkyLookupAutocompleteAdapter {
    * The function to dynamically manage the data source when users
    * change the text in the lookup field. The search function must return
    * an array or a promise of an array. The `search` property is particularly
-   * useful when the data source does not live in the source code. If the
-   * search requires calling a remote data source, use `searchAsync` instead of
-   * `search`.
+   * useful when the data source does not live in the source code.
+   * @deprecated The `searchAsync` event emitter and callback to provide searched data to the lookup component.
```

Comment:
Should these comments be `@deprecated Use the searchAsync...` instead?

Reply:
Good catch. Fixed.